.travis.yml

```diff
@@ -75,17 +75,17 @@ matrix:
         - make ${BUILDTYPE}
       # Overrides `script` to disable publishing
       script: true
-    # Coverage build
-    - os: linux
-      env: BUILDTYPE=debug CXXFLAGS="--coverage" LDFLAGS="--coverage"
-      # Overrides `script` to publish coverage data to codecov
-      script:
-        - export PATH=$(pwd)/mason_packages/.link/bin/:${PATH}
-        - PATH=`echo $PATH | sed -e 's/:\.\/node_modules\/\.bin//'` # remove relative path ^^
-        - which llvm-cov
-        - curl -S -f https://codecov.io/bash -o codecov
-        - chmod +x codecov
-        - ./codecov -x "llvm-cov gcov" -Z
+    # # Coverage build
```

Comment:
Was this removed because of the codecov security breach or because it doesn't work any more or for some other reason?

Reply:
Yes it was! Sorry that wasn't clear - I'm mostly cribbing from https://github.com/mapbox/vtcomposite/pull/111/files